### PR TITLE
Mitigations against event type manipulation in UEFI eventlog

### PR DESCRIPTION
**Pull request by THS-on**
This provides the initial hardening of the example policy against event type manipulation. 
This not directly an issue with Keylime, because the UEFI eventlog does not include the metadata in the hash that is used to extend the PCR. The original issue can be found here: https://github.com/google/go-attestation/blob/master/docs/event-log-disclosure.md

The most effective way to mitigate against that is to reduce the usage of `acceptAll ` and limit how often they can be used.

As discussed with @mpeters we will first update our example policy and then provide documentation on how to write custom policies.

### CODE REVIEW

**Comment by THS-on** [comment]
@kkaarreell the Fedora rawhide tests are currently failing. Is there something that we can fix them or can we just disable them until the issue in rawhide itself is solved?

**Comment by kkaarreell** [comment]
According to @sergio-correia same patches as in
 https://bugzilla.redhat.com/show_bug.cgi?id=1984634
are missing in rawhide.
Sergio, did you open a rawhide bug regarding it?

Re tests, do you mean disable them all or just on Rawhide?

**Comment by sergio-correia** [comment]
@kkaarreell: there was a bug open for it already [1], so I just added a comment there some days ago. The maintainer of `tpm2-tss` in Fedora is currently waiting for a newer release -- to be released soon(tm) --, which should support OpenSSL 3.

[1] https://bugzilla.redhat.com/show_bug.cgi?id=2008179

**Comment by kkaarreell** [comment]
@THS-on Would you mind adding a commit that replaces `fedora-all` with `fedora-stable` in `. packit. yaml`?

**Comment by THS-on** [comment]
@kkaarreell done. The tests now pass.

**Comment by mpeters** [comment]
@maugustosilva any chance you could take a look at these changes?

**Review by maugustosilva — APPROVED** [review]

**Comment by maugustosilva** [comment]
Passes all my tests, let's merge

**Pull request merged**
